core/src/test/java/org/elasticsearch/search/aggregations/BaseAggregationTestCase.java

```diff
@@ -80,7 +80,7 @@
 import static org.elasticsearch.cluster.service.ClusterServiceUtils.setState;
 import static org.hamcrest.Matchers.equalTo;
 
-public abstract class BaseAggregationTestCase<AB extends AggregatorBuilder<AB>> extends ESTestCase {
+public abstract class BaseAggregationTestCase<AB extends AbstractAggregatorBuilder<AB>> extends ESTestCase {
```

Comment:
The tests use the getType() method which is only available on AbstractAggregatorBuilder.